packages/eslint-plugin-pf-codemods/lib/rules/v5/datepicker-warn-helperText.js

```diff
@@ -0,0 +1,30 @@
+const { getPackageImports } = require("../../helpers");
+
+  // https://github.com/patternfly/patternfly-react/pull/8833
+  module.exports = {
+    meta: {},
+    create: function (context) {
+      const imports = getPackageImports(
+        context,
+        "@patternfly/react-core"
+      ).filter((specifier) => specifier.imported.name === "DatePicker");
```

Comment:
no biggie but as you are just getting a single thing here you can use `find` and then not have to map it out below